src/main/java/org/mapstruct/intellij/inspection/TargetPropertyMappedMoreThanOnceInspection.java

```diff
@@ -0,0 +1,210 @@
+                    .forEach( target ->
+                            problemMap.computeIfAbsent( target, k -> new ArrayList<>() ).add( psiAnnotation ) );
+        }
+
+        private static void handleMappingAnnotation(PsiAnnotation psiAnnotation,
+                                                    Map<String, List<PsiElement>> problemMap) {
+            PsiAnnotationMemberValue value = psiAnnotation.findDeclaredAttributeValue( "target" );
+            if (value != null) {
+                String target = getStringAttributeValue( value );
+                if (target != null) {
+                    problemMap.computeIfAbsent( target, k -> new ArrayList<>() ).add( value );
+                }
+            }
+        }
+
+        private static class ChangeTargetQuickFix extends LocalQuickFixOnPsiElement {
+
+            private final String myText;
+            private final String myFamilyName;
+
+            private ChangeTargetQuickFix(@NotNull PsiAnnotationMemberValue element) {
+                super( element );
+                myText = MapStructBundle.message( "intention.change.target.property" );
+                myFamilyName = MapStructBundle.message( "inspection.target.property.mapped.more.than.once",
+                        element.getText() );
+            }
+
+            @Override
+            public @IntentionName @NotNull String getText() {
+                return myText;
+            }
+
+            @Override
+            public void invoke(@NotNull Project project, @NotNull PsiFile psiFile, @NotNull PsiElement psiElement,
+                               @NotNull PsiElement psiElement1) {
+                FileEditor selectedEditor = FileEditorManager.getInstance( project ).getSelectedEditor();
+                if ( selectedEditor instanceof TextEditor) {
+                    Editor editor = ( (TextEditor) selectedEditor ).getEditor();
+
+                    TextRange textRange = psiElement.getTextRange();
+                    String textOfElement = String.valueOf( editor.getDocument()
+                            .getCharsSequence()
+                            .subSequence( textRange.getStartOffset(), textRange.getEndOffset() ) );
+                    int targetStart = Strings.indexOf( textOfElement, "\"" ) + 1;
+                    int targetEnd = textOfElement.lastIndexOf( "\"" );
+
+                    editor.getCaretModel().moveToOffset( textRange.getStartOffset() + targetStart );
+                    LogicalPosition startPosition = editor.getCaretModel().getLogicalPosition();
+                    editor.getCaretModel().moveToOffset( textRange.getStartOffset() + targetEnd );
+                    editor.getCaretModel().setCaretsAndSelections(
+                            Collections.singletonList( new CaretState(startPosition, startPosition,
+                                    editor.getCaretModel().getLogicalPosition() ) ) );
+                    editor.getScrollingModel().scrollToCaret( ScrollType.MAKE_VISIBLE );
```

Comment:
```suggestion
                    WriteCommandAction.runWriteCommandAction(project, () ->
                        editor.getDocument().replaceString(textRange.getStartOffset(), textRange.getEndOffset(), "\"\"")
                    );
                    editor.getCaretModel().moveToOffset( textRange.getStartOffset() + 1 );

```

Another possibility would be to remove the text and set the cursor into the empty String `""` like this:
Before:
![before](https://github.com/user-attachments/assets/bf6ed274-e76b-4362-b00f-b8d0ab52ba3e)

After:
![after](https://github.com/user-attachments/assets/8fce6f7f-f4a8-4886-9b33-bfa3996d9a24)

Just another suggestion, don't know if that is better. At least users can immediately press `Ctrl + Space` to get the correct auto complete suggestions.

Reply:
The same works if the the text is just selected. Maybe there is a way to start the suggestion automatically.

Reply:
Haven't found an easy way. It seems to be possible by using internal services, which I wouldn't recommend to do, see here: https://intellij-support.jetbrains.com/hc/en-us/community/posts/360004151660-Get-current-auto-complete-at-cursor

My motivation for removing the text was a test case where the selected text does not start like the completion proposals. In my screenshot there is only the second property "seatCount" which gets only proposed when I delete the text as it does not start with "maker*".

But I think it's fine if only the text is selected.

Leaving this unresolved, maybe @filiphr has an opinion about it.

Reply:
I don't have any strong preference. Seems like only selecting the entire text and pressing the auto complete triggers the auto complete as if there is no text.

Reply:
> Seems like only selecting the entire text and pressing the auto complete triggers the auto complete as if there is no text.

That didn't work for me last time, but in fact it works now. Another reason why the text selection should be sufficient.

Reply:
I would keep the text selection option. If someone wants to change to target to a inner target, it might help that the old target stayed.